controllers/quickstarts_test.go

```diff
@@ -72,7 +83,7 @@ func TestEnsureQuickStarts(t *testing.T) {
 	assert.NoError(t, err)
 	err = fakeReconciler.Client.Create(context.TODO(), &testQuickStartCRD)
 	assert.NoError(t, err)
-	err = fakeReconciler.ensureQuickStarts(fakeReconciler.Log)
```

Comment:
Why can not you directly call the func?

Reply:
I previously had some other logic in there before this. I've fixed it now